projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_base.py

```diff
@@ -120,7 +121,7 @@ def __init__(
 
     def send_object_for_ingestion(
         self,
-        payload: dict,
+        payload: Union[dict, "pandas.DataFrame"],
```

Comment:
Make sure not to forget upating https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/src/vdk/api/job_input.py#L140 which is our "public" doc. 

Also of course don't forget a test. Not that I think you would .

Reply:
besides for these two notes, do you think everything's ok with calling send_tabular_data from inside send_object? I'm worried that this might create some maintenance issues in the future but it's possible I'm overestimating the likelihood of that